docs/_static/admonitions.css

```diff
@@ -0,0 +1,47 @@
+/* note box */
+.rst-content .note .admonition-title {
+    border-top-right-radius: 3px;
+    border-top-left-radius: 3px;
+    background-color: var(--blue);
+}
+
+.rst-content .note {
+    border-radius: 3px;
+    background-color: var(--mediumblue);
+}
+
+.rst-content .note a {
+    color: var(--darkblue);
+}
+
+/* warning box */
+.rst-content .warning .admonition-title {
+    border-top-right-radius: 3px;
+    border-top-left-radius: 3px;
+    background-color: var(--darkpink);
+}
+
+.rst-content .warning {
+    border-radius: 3px;
+    background-color: var(--lightpink);
+}
+
+.rst-content .warning a {
+    color: var(--darkpink);
+}
+
+/* important box */
+.rst-content .important .admonition-title {
+    border-top-right-radius: 3px;
+    border-top-left-radius: 3px;
+    background-color: var(--darkorange);
+}
+
+.rst-content .important {
+    border-radius: 3px;
+    background-color: var(--lightorange);
+}
+
+.rst-content .important a {
+    color: var(--darkorange);
+}
```

Comment:
eolrip

Reply:
the formatter in vs code removes blank lines at the end of files. I left it because if I make changes in the future this will keep happening. can add them if you think it's better

Reply:
Ah okay, then leave it as is. It's not like we look at these files every day anyhow.